rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -1289,24 +1287,22 @@ bool game_list_frame::RemovePPUCache(const std::string& base_dir, bool is_intera
 	u32 files_removed = 0;
 	u32 files_total = 0;
 
-	QDirIterator dir_iter(qstr(base_dir), QDir::Files | QDir::NoDotAndDotDot, QDirIterator::Subdirectories);
+	QDirIterator dir_iter(qstr(base_dir), QStringList() << QStringLiteral("*.obj") << QStringLiteral("*.obj.*"), QDir::Files | QDir::NoDotAndDotDot, QDirIterator::Subdirectories);
```

Comment:
Maybe mark it explicitly as .obj.gz, otherwise it looks like it may take unrelated files